k8s/wordpress/chart/wordpress/templates/application.yaml

```diff
@@ -129,9 +129,9 @@ spec:
     kind: Secret
   - group: v1
     kind: Service
-  {{ if .Values.enablePublicServiceAndIngress }}
+  {{- if .Values.enablePublicServiceAndIngress }}
```

Comment:
Can you educate me why this change is needed?

Reply:
Found my answer at https://helm.sh/docs/chart_template_guide/control_structures/

Reply:
Yes :) It avoids white spaces if the var is false .